### PR TITLE
[SDK-637] Support connecting to FSS

### PR DESCRIPTION
**Pull request by jdm717**
## What

Splits our viewer SDK into two separate packages, being `viewer` (for the platform and connecting to FSS) and `viewer-eedc` (for eedc and connection to ISS). The `viewer` package currently supports loading a scene, but no further interaction, and as such, most of the contents have been removed for now. To be added back:
- View cube
- Viewer toolbar
- Scene manipulation

A note with these changes, all the `viewer-eedc` files have just been moved over from the `viewer` package, [this diff](https://github.com/Vertexvis/vertex-web-sdk/pull/5/files#diff-1868c137ba346e192210526da2d086fcL34) is the start of changes to the `viewer` package that reflect the actual additions in this PR

## Testing 

- Using the `viewer` SDK
  - Provide a scene id through `viewer.load('urn:vertexvis:scene:{id}')`
  - Configure credentials for platdev
  - Verify an image is rendered

### CODE REVIEW

File: packages/viewer/src/components/viewer/viewer.tsx [hunk 1]
```diff
@@ -429,57 +357,16 @@ export class Viewer {
    *  * `urn:vertexvis:eedc:file:<fileid>`
    *  * `urn:vertexvis:eedc:scenestate:<scenestateid>`
    *  * `urn:vertexvis:eedc:file?externalId=<externalId>`
+   *  * `urn:vertexvis:platform:scene:<sceneid>`
```
Comment (jdm717) [inline packages/viewer/src/components/viewer/viewer.tsx:360]: Update this comment

File: packages/viewer/src/components/viewer/viewer.tsx [hunk 2]
```diff
@@ -506,6 +395,38 @@ export class Viewer {
...
+
+          throw new ExpiredCredentialsError(
+            'Error loading model. Could not open websocket due to the provided credentials being expired.',
+            e
+          );
+        } else {
+          this.errorMessage = 'Error loading model. Could not open websocket.';
+          throw new WebsocketConnectionError(
+            'Error loading model. Could not open websocket.',
+            e
```
Comment (jdm717) [inline packages/viewer/src/components/viewer/viewer.tsx:417]: model -> scene

File: packages/viewer/src/components.d.ts [hunk 3]
```diff
@@ -50,26 +47,17 @@ export namespace Components {
...
-        "getFrameAttributes": () => Promise<FrameAttributes>;
+        "getFrameAttributes": () => Promise<FrameAttributes.FrameAttributes | undefined>;
         "getInteractionHandlers": () => Promise<InteractionHandler[]>;
         /**
           * @private Used internally for testing.
          */
         "httpClient": HttpClient.HttpClient;
         /**
-          * Loads the given resource into the viewer and return a `Promise` that resolves when the scene has been loaded. The specified resource is a URN in one of the following formats:    * `urn:vertexvis:eedc:file:<fileid>`   * `urn:vertexvis:eedc:scenestate:<scenestateid>`   * `urn:vertexvis:eedc:file?externalId=<externalId>`
+          * Loads the given resource into the viewer and return a `Promise` that resolves when the scene has been loaded. The specified resource is a URN in one of the following formats:    * `urn:vertexvis:eedc:file:<fileid>`   * `urn:vertexvis:eedc:scenestate:<scenestateid>`   * `urn:vertexvis:eedc:file?externalId=<externalId>`   * `urn:vertexvis:platform:scene:<sceneid>`
```
Comment (danconner-vertex) [inline packages/viewer/src/components.d.ts:57]: as discussed, remove `platform`

File: packages/viewer/src/components/viewer/readme.md [hunk 4]
```diff
@@ -59,25 +59,14 @@ one of the following formats:
   * `urn:vertexvis:eedc:file:<fileid>`
   * `urn:vertexvis:eedc:scenestate:<scenestateid>`
   * `urn:vertexvis:eedc:file?externalId=<externalId>`
+  * `urn:vertexvis:platform:scene:<sceneid>`
```
Comment (danconner-vertex) [inline packages/viewer/src/components/viewer/readme.md:62]: as discussed, remove `platform`

File: packages/viewer/src/components/viewer/viewer.spec.ts [hunk 5]
```diff
@@ -238,13 +221,53 @@ describe('vertex-viewer', () => {
...
+    it('loads a platform scene', async () => {
+      const viewer = await createViewerSpec(`<vertex-viewer></vertex-viewer`);
+      const startPromise = new Promise(resolve => {
+        viewer.registerCommand('stream.start', dimensions => {
+          return ({ stream }) => {
+            resolve(dimensions);
+          };
+        });
+      });
+      await viewer.load('urn:vertexvis:platform:scene:scene-id');
```
Comment (danconner-vertex) [inline packages/viewer/src/components/viewer/viewer.spec.ts:242]: as discussed, remove `platform`

**Comment by jdm717** [comment]
@danconner-vertex this PR has been updated and should be ready for another look when you get the chance

**Pull request merged**